crates/rpc/rpc-eth-types/src/error.rs

```diff
@@ -94,6 +94,33 @@ pub enum EthApiError {
     /// Thrown when a requested transaction is not found
     #[error("transaction not found")]
     TransactionNotFound,
+    /// Thrown when max depth is reached
+    #[error("max depth reached")]
+    MaxDepth,
+    /// Thrown when a bundle is unmatched
+    #[error("unmatched bundle")]
+    UnmatchedBundle,
+    /// Thrown when a bundle is too large
+    #[error("bundle too large")]
+    BundleTooLarge,
+    /// Thrown when validity is invalid
+    #[error("invalid validity")]
+    InvalidValidity,
+    /// Thrown when inclusion is invalid
+    #[error("invalid inclusion")]
+    InvalidInclusion,
+    /// Thrown when a bundle is invalid
+    #[error("invalid bundle")]
+    InvalidBundle,
+    /// Thrown when a bundle simulation times out
+    #[error("bundle simulation timed out")]
+    BundleTimeout,
+    /// Thrown when a transaction is reverted in a bundle
+    #[error("bundle transaction failed")]
+    BundleTransactionFailed,
+    /// Thrown when a bundle simulation returns negative profit
+    #[error("bundle simulation returned negative profit")]
+    NegativeProfit,
```

Comment:
these are all bundle specific errors, I'd rather have a new standalone error type for this

Reply:
resolved in 2dce6a4